server/internal/app-config/config.go

```diff
@@ -54,21 +54,24 @@ var (
 )
 
 type Config struct {
-	DbType              string
-	EventDbType         string
-	DbDir               string
-	DbMigrationPath     string
-	EventDbDir          string
-	RoundInterval       int64
-	Network             common.Network
-	SchedulerType       string
-	TxBuilderType       string
-	WalletAddr          string
-	RoundLifetime       int64
-	UnilateralExitDelay int64
-	BoardingExitDelay   int64
-	NostrDefaultRelays  []string
-	NoteUriPrefix       string
+	DbType                  string
+	EventDbType             string
+	DbDir                   string
+	DbMigrationPath         string
+	EventDbDir              string
+	RoundInterval           int64
+	Network                 common.Network
+	SchedulerType           string
+	TxBuilderType           string
+	WalletAddr              string
+	RoundLifetime           int64
+	UnilateralExitDelay     int64
+	BoardingExitDelay       int64
+	NostrDefaultRelays      []string
+	NoteUriPrefix           string
+	MarketStartTime         int64
```

Comment:
Let's keep the same prefix for consitency:
```suggestion
	MarketHourStartTime         int64
```